evaluator/evaluator_test.go

```diff
@@ -1020,16 +1016,12 @@ func TestBuiltinProperties(t *testing.T) {
 	}{
 		{`"a".ok`, false},
 		{`"a".inv`, "invalid property 'inv' on type STRING"},
-		{"a = $(date);\na.ok", true},
+		{"a = cd();\na.ok", true},
```

Comment:
we should keep a test that checks for `cmd.ok` as `cd` is a special case on its own